libsrc/boblightserver/BoblightClientConnection.cpp

```diff
@@ -88,10 +116,17 @@ void BoblightClientConnection::socketClosed()
 	emit connectionClosed(this);
 }
 
+
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
+#define SkipEmptyParts (Qt::SkipEmptyParts)
+#else
+#define SkipEmptyParts (QString::SkipEmptyParts)
+#endif
+
 void BoblightClientConnection::handleMessage(const QString & message)
 {
 	//std::cout << "boblight message: " << message.toStdString() << std::endl;
-	QStringList messageParts = QStringUtils::split(message," ",QStringUtils::SplitBehavior::SkipEmptyParts);
+	const QVector<QStringRef> messageParts = message.splitRef(' ', SkipEmptyParts, Qt::CaseSensitive);
```

Comment:
You might want using the split wrapper from to move away from defines:
https://github.com/hyperion-project/hyperion.ng/blob/master/include/utils/QStringUtils.h

Reply:
Extended QStringUtils to support `splitRef` in addition to `split` in 014e0bb3633bb6ca1ecd96d3aea048f49ca29b26 .